src/app/ui/dithering_selector.cpp

```diff
@@ -255,9 +262,9 @@ void DitheringSelector::regenerate()
       }
       break;
   }
-
-  setSelectedItemIndex(0);
-  setSizeHint(getItem(0)->sizeHint());
+  selectedItemIndex = std::min(std::max(0, selectedItemIndex), getItemCount()-1);
```

Comment:
```suggestion
  selectedItemIndex = base::clamp(selectedItemIndex, 0, std::max(0, getItemCount()-1));
```
We need to include "base/clamp.h" for this.

Reply:
I used `std::clamp` instead.

Reply:
Oh yes, now we can use it 👍